libr/util/r_asn1.c

```diff
@@ -10,6 +10,19 @@
 
 const char* _hex = "0123456789abcdef";
 
+static char* sanitize(char *s, ut32 len) {
+	if (!s) {
+		return NULL;
+	}
+	char* e = s;
+	while (s <= (e + len)) {
+		if(*s < 0x20 || *s > 0x7E)
```

Comment:
actually no. i've checked, and no code does ASCII only.
if yes, i'll be happy to remove it.